src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertySymbolBase.cs

```diff
@@ -1579,7 +1579,10 @@ protected override void DecodeWellKnownAttributeImpl(ref DecodeWellKnownAttribut
                 }
                 else
                 {
-                    diagnostics.Add(ErrorCode.ERR_UnscopedRefAttributeUnsupportedMemberTarget, arguments.AttributeSyntaxOpt.Location);
+                    var code = this.GetMethod?.UseUpdatedEscapeRules == false || this.SetMethod?.UseUpdatedEscapeRules == false
```

Comment:
These are source accessors, so they just delegate to the containing module. Perhaps it would be simpler to `this.ContainingModule.UseUpdatedEscapeRules` instead.